### ISSUE TITLE
No Diff highlight on latest vscode-1.48.0

### ISSUE THREAD

**Issue by shashikiran-sk** [stef-levesque/vscode-perforce#193]
vscode not showing the code diff highlight bars when editing the file in any changelist.
To know the code diff have to open diff against revision view.

**Comment by davestevens** [comment]
Yeah, this is something that I use all the time. Seems that the "print -q" isn't being called when opening a file like it was in the previous version. Looks like there were changes to dirtydiffDecorator.ts in vscode between the two releases.
Can't seem to figure out how to actually debug vscode internals to take a look though.

**Comment by davestevens** [comment]
This has hit other scm providers: https://github.com/microsoft/vscode/issues/104786

**Comment by admvx** [comment]
I have switched over to this fork/rewrite, which works well on 1.48.0: https://marketplace.visualstudio.com/items?itemName=mjcrouch.perforce

(Related discussion with this extension's author on the current divergence: https://github.com/mjcrouch/vscode-perforce/issues/135)
